designdocs/batched_messages.md

```diff
@@ -62,6 +62,86 @@ See the [implementation of paging in the messages pallet](https://github.com/Lib
     * `next_index`: `u32` starting index of next results in `next_block`
     * `results`: `Vec<BatchAnnouncement>`
 
+### Message Schema
+
+Distributing data on a decentralized network has a few fundamental problems that
+are worth discussing here. Primarily, those problems are:
+- What shape is this data in?
+- How do I authenticate this data?
+- How do these actors access this information?
+
+Less primary but very much salient are a couple of secondary concerns:
+- How do I make sure this data occupies no more space / time than necessary?
+- How do I make sure this data can be easily indexed and searched?
+
+In the days of yore, it was easy to answer these questions. A centralized
+service would declare an output shape and make it available to the world via an
+API they would maintain and authorize. In our world, it is not so simple. Each
+actor on a DSNP-implementing system must be able to imbue their announcements
+with trust without exerting control over the entire network. Announcements can
```

Comment:
Will `Announcent` need to be modified to match the nomenclature defined in the Frequency Chain diagram or is this related to DSNP spec?